### ISSUE TITLE
mp4 with multi audio support?

### ISSUE THREAD

**Issue by hiren3897** [shaka-project/shaka-player#4933]
**Have you read the [Tutorials](http://shaka-player-demo.appspot.com/docs/api/tutorial-welcome.html)?**
YES

**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
YES

**What version of Shaka Player are you using?**
v4.3.1

**Please ask your question**
Does shaka player support multi-audio with `.mp4` content

I tried to play an mp4 video content with multi-audio but the shaka player doesn't detect the languages.

is there any way to detect the audio languages since the shaka player plays the `.mp4` content on the `src=` attribute which is handled by the browser could be the issue? 

URL: https://mirror.eu.oneandone.net/projects/media.ccc.de/congress/2019/h264-hd/36c3-10496-deu-eng-spa-Das_Mauern_muss_weg_hd.mp4

**Comment by hiren3897** [comment]
is it possble to add an Audio Track? for an mp4 file?

**Comment by joeyparrish** [comment]
Playing a single mp4 delegates to the browser, which doesn't give us any way of switching audio tracks AFAICT.  If you play that mp4 URL directly in Chrome, the native controls don't offer to change languages or audio tracks, and Chrome has not implemented the `.audioTracks` property on the video element.

Sorry, but there's nothing we can do for this at the player level.  You can file bugs on Chrome at https://crbug.com/ to ask them to implement these features.

**Comment by hiren3897** [comment]
Yes @joeyparrish 

Thanks for the information

I also did some research and what I found is

https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/audioTracks#browser_compatibility

Audio tracks are supported in the Shaka Player MSE playback engine for HLS and DASH. Safari also supports audio tracks for native playback of HLS and MP4. Other browsers do not support audio tracks for native playback including MP4, so it is impossible to work with MP4 audio tracks in Chrome or Firefox for example.

- CHROME
  - There are some experimental flags in chrome
    `#enable-experimental-web-platform-features` preferences (needs to be set to Enabled). To change preferences in Chrome, visit chrome://flags. `I tested this we are able to see the languages but the behavior is not good

- FIREFOX
  - From version 33: this feature is behind the `media.track.enabled` preferences (needs to be set to true). To change preferences in Firefox, visit about config. `doesn't work perfectly


Some helpful docs
https://jeremyrichardson.dev/blog/multiple_audio_tracks_in_chrome
https://bugzilla.mozilla.org/show_bug.cgi?id=1119299

Thanks :)
